src/Parser.php

```diff
@@ -40,7 +40,7 @@ private function parseParameters($text)
         $return = array();
         foreach($argsMatches[1] as $item)
             {
-            $parts = explode($this->syntax->getParameterValueSeparator(), $item, 2);
+            $parts = preg_split('~\s*' . $this->syntax->getParameterValueSeparator() . '\s*~', $item, 2);
```

Comment:
I prefer to not add the spaces around concatenation. AFAIK PSR-2 does not enforce that. Also do we need `preg_split()`? Won't `trim()`ming be enough?